web-api/src/models.rs

```diff
@@ -106,15 +119,12 @@ pub(crate) struct PersonalizedDocument {
     pub(crate) score: f32,
 
     /// Embedding from smbert.
-    #[serde(skip_serializing)]
     pub(crate) embedding: Embedding,
 
     /// Contents of the document properties.
-    #[serde(default)]
     pub(crate) properties: DocumentProperties,
 
     /// The tags associated to the document.
-    #[serde(default)]
```

Comment:
I'm not sure if removing the default is the best idea here having it a bit more robust by default for a non-fixed schema backend like elastic search is a good idea even if we don't  take advantage of it.